porady_odnosnie_studiowania.md

```diff
@@ -348,3 +348,93 @@ Choć w poprzednim punkcie dość dużo wagi położyliśmy na to, żeby nie ba
+
+  <details>
+    <summary>Szczegóły</summary>
+
+    Pracę dyplomową najlepiej pisać na temat na którym się znasz, który Cię interesuje i z którym wiążesz swoją przyszłość. Przede pomoże Ci to pracować nad nią systematycznie i przemóc ewentualną prokrastynację. Ponadto najpewniej zamieścisz tytuł i opis pracy (a może nawet repozytorium pod którym można znaleźć jej treść/kod napisany w jej ramach) w swoim CV/portfolio/liście intencyjnym/etc. - dlatego warto, żeby praca była najwyższej możliwej jakości w czym chęci do tworzenia jej zdecydowanie pomagają.
+  </details>
+
+* **Pracuj systematycznie i zacznij wcześnie**
+
+  <details>
+    <summary>Szczegóły</summary>
+
+    Jakkolwiek kuszące może być odłożenie robienia pracy dyplomowej na ostatnią chwilę, jest to bardzo zła praktyka która źle odbije się na jej jakości, Twoim samopoczuciu i finalnej ocenie. Najlepszą strategią jest regularna pisanie pracy i jej komponentów, począwszy od początku semestru w który musisz ją oddać. Temat pracy i jej promotora warto wybrać jeszcze wcześniej.
+  </details>
+
+* **Promotor nie napisze za Ciebie pracy**
+
+  <details>
+    <summary>Szczegóły</summary>
+
+    Promotor w procesie pisania pracy dyplomowej jest przewodnikiem oraz oceniającym a nie jednym z jej autorów. Możesz zwrócić się do niego z pytaniami dotyczącymi np. technologii które chcesz wykorzystać czy propozycji rozwiązania/obejścia problemu na który się natknąłeś. Nie licz natomiast, że promotor będzie prowadził Cię za rękę lub pilnował czy pracujesz systematycznie.
+
+    Ponadto powinieneś pamiętać, że Twój promotor najpewniej zajmuje się kilkoma pracami dyplomwymi naraz i każdej musi poświęcić uwagę.
+  </details>
+
+* **Pamiętaj o terminach**
+
+  <details>
+    <summary>Szczegóły</summary>
+
+    Praca dyplomowa musi zostać wprowadzona do systemu APD (Archiwum Prac Dyplomowych) na minimum tydzień przed terminem prezentacji/obrony. Jednakże w systemie musi zostać zatwierdzona przez promotora a potem przepuszczona przez system antyplagiatowy (który potrafi sprawdzać pracę nawet do 2-3 dni), dlatego warto zamieścić ją w APD kilka dni przed obowiązkowym terminem.
+
+    Ponadto pracę przed zamieszczeniem w APD czyta promotor i zwraca do niej zestaw uwag do których musisz się odnieść i poprawić wskazane w nich fragmenty. Warto zarezerwować na ten proces jeden do dwóch tygodni, zwłaszcza że praca może być odesłana do poprawki przez promotora kilkakrotnie.
+  </details>
+
+* **Przygotuj się do obrony/prezentacji**
+
+  <details>
+    <summary>Szczegóły</summary>
+
+    Ocena pracy dyplomowej składa się z oceny samej pracy oraz oceny jej prezentacji/obrony przed promotorem oraz recenzentem. Dlatego ważne jest, aby dobrze przygotować się do zaprezentowania pracy.
+
+    Koniecznie przyjdź na obronę z własnym sprzętem - laptopem do którego można podłączyć standardowy wydziałowy rzutnik za pomocą kabla HDMI (kabel i rzutnik będą na Ciebie czekały).
+
+    Prezentacja pracy składa się z dwóch części:
```

Comment:
W przypadku pracy magisterskiej jest jeszcze egzamin magisterski ustny. Być może warto to jakoś zaznaczyć, albo dopisać że chodzi o prace licencjackie.